vendor/bat-native-ledger/src/bat_publishers.cc

```diff
@@ -903,6 +903,8 @@ void BatPublishers::setBalanceReportItem(ledger::ACTIVITY_MONTH month,
 
 void BatPublishers::getPublisherBanner(const std::string& publisher_id,
                                        ledger::PublisherBannerCallback callback) {
+
+  BLOG(ledger_, ledger::LogLevel::LOG_ERROR) << "NEJC getPublisherBanner";
```

Comment:
Looks like logs were left in on this file

Reply:
removed